diff --git a/src/org.gnome.OfficeRunner.appdata.xml.in b/src/org.gnome.OfficeRunner.appdata.xml.in
index 835f054..796f86f 100644
--- a/src/org.gnome.OfficeRunner.appdata.xml.in
+++ b/src/org.gnome.OfficeRunner.appdata.xml.in
@@ -2,13 +2,12 @@
 <!-- Copyright 2014 William Moreno Reyes <williamjmorenor@gmail.com>
      Copyright 2014 Bastien Nocera <hadess@hadess.net> -->
 
-<component type="desktop">
-  <id type="desktop">org.gnome.OfficeRunner.desktop</id>
+<component type="desktop-application">
+  <id>org.gnome.OfficeRunner.desktop</id>
   <name>Office Runner</name>
   <metadata_license>CC0-1.0</metadata_license>
-  <summary>Close your laptop lid and start running</summary>
-  <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
+  <summary>Close your laptop lid and start running</summary>
   <description>
     <p>
       Office Runner is a game where tallying up the time taken to go from one place in
@@ -20,15 +19,24 @@
       short trip.
     </p>
   </description>
-  <url type="homepage">http://www.hadess.net/2011/09/omg-i-haz-designed-bug-fix.html</url>
+  <url type="homepage">https://www.hadess.net/2011/09/omg-i-haz-designed-bug-fix.html</url>
+  <url type="bugtracker">https://gitlab.gnome.org/GNOME/office-runner/-/issues</url>
+  <url type="vcs-browser">https://gitlab.gnome.org/GNOME/office-runner/</url>
+  <url type="translate">https://l10n.gnome.org/module/office-runner/</url>
   <screenshots>
-    <screenshot type="default">https://git.gnome.org/browse/office-runner/plain/src/office-runner-appdata-screenshot.jpg</screenshot>
+    <screenshot type="default">
+      <image type="source">https://gitlab.gnome.org/GNOME/office-runner/raw/master/src/office-runner-appdata-screenshot.jpg</image>
+    </screenshot>
   </screenshots>
   <releases>
     <release version="1.0.3" date="2018-10-24"/>
   </releases>
-  <updatecontact>hadess@hadess.net</updatecontact>
+  <update_contact>hadess@hadess.net</update_contact>
+  <developer id="net.hadess">
+    <name translate="no">Bastien Nocera</name>
+  </developer>
   <project_group>GNOME</project_group>
   <translation type="gettext">office-runner</translation>
+  <launchable type="desktop-id">org.gnome.OfficeRunner.desktop</launchable>
   <content_rating type="oars-1.0" />
 </component>
